int64   frame

# Black strip
bool    black_strip_flag
uint32  black_strip_x
uint32  black_strip_y